frontends/beta-frontend/shared/components/Grid/Grid.tsx

```diff
@@ -9,7 +10,36 @@ type GridProps<T> = {
 
 function Grid<T>({ rows, columns }: GridProps<T>) {
   return (
-    <Box>
+    <Box sx={{
```

Comment:
did you see any example with styling in that way somewhere?

Reply:
Yes, it was one of the examples in the material ui documentation: 
https://mui.com/x/react-data-grid/style/
Here's the codesandbox: https://codesandbox.io/s/5c5rqy?file=/src/Demo.tsx

I was debating to create the css clases outside the Box component with emotion, do you have any suggestions?